src/targets/base.ts

```diff
@@ -62,6 +62,12 @@ export class BaseTarget {
       ...defaultFilterOptions,
       ...this.filterOptions,
     };
+    if (!filterOptions.includeNames) {
```

Comment:
I'm pretty sure this will break some repos, because `includeNames` is usually undefined.
We can probably do a `filterOptions.includeNames ===  []` comparison, so `[]` in `includeNames` can have an explicit meaning of "do not include anything".